index.html

```diff
@@ -20,7 +21,8 @@ <h2> A Timeline of Jerry Lawson's life </h2>
           <li> In the 1970s, his interest in computing encouraged him into Silicon Valley's Homebrew Computer Club, where he was the only black member at the time and eventually crossed paths with Steve Jobs and Steve Wozniak. He referred to Jobs as a business-minded "sparkplug" and was unimpressed when he interviewed Wozniak for a job.
           <li> In the mid-1970s, Lawson helped create the Fairchild Channel F, which was produced in 1976 by Fairchild Semiconductor, where he worked as director of engineering and marketing.
           <li> His work allowed people to play a variety of games in their homes and paved the way for systems, such as the Atatri 2600, Nintendo, Xbox and Playstation.
-          <li> On April 9, 2011, Jerry Lawson died in Mountain View, California due to complications relating to his diabetes. He was survived by his wife, Catherine, and his two children. 
+          <li> On April 9, 2011, Jerry Lawson died in Mountain View, California due to complications relating to his diabetes. He was survived by his wife, Catherine, and his two children.
+        </ul>
```

Comment:
I added a closing`</ul>` tag here to make sure we close the `<ul>` tag on line 17.

Reply:
Can't believe I miss those! Thanks!

Reply:
I forgot closing </li> tags!